src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -213,6 +215,7 @@
                   <matches target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='isa-date']/@value" datatype="date-time"/>
                   <matches target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='ipv4-address']/@value" datatype="ip-v4-address" />
                   <matches target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='ipv6-address']/@value" datatype="ip-v6-address" />
+                  <matches target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='uri']/@value" datatype="uri" />
```

Comment:
This was not explicitly requested in https://github.com/usnistgov/OSCAL/issues/1913#issuecomment-2484136304, but it standards to reason we make sure it is a URI, not just a string value for a prop with the `.[@name="uri"]`. Open question: the documentation string defines this as a URI, we are sure we do not mean a URL to access the interconnect with a given scheme/authority, host, path, or is it really _just_ an identifier. I think some (Brian and Michaela know this detail), but URIs and URLs are related in terms of type signature. In Metaschema, they would use the same type.